#Virtual Guidance, Axis Type
virtual_guidance:
  position:
    x: 0.0
    y: 0.1 #[m]
    z: 0.0
  stiffness: 5 # N/mm
  damping: 0
